### ISSUE TITLE
function pointer inside FieldDecl

### ISSUE THREAD

**Issue by Konstantin8105** [elliotchance/c2go#500]
add function pointer inside FieldDecl
```
// Warning (FieldDecl): 1248: function pointers are not supported [2] : 'void ( *(*)(sqlite3_vfs *, void *, char *))(void)'
// Warning (FieldDecl): 1263: function pointers are not supported [1] : 'int (*)(sqlite3_vfs *, char *, sqlite3_syscall_ptr)'
// Warning (FieldDecl): 1264: function pointers are not supported [1] : 'sqlite3_syscall_ptr (*)(sqlite3_vfs *, char *)'
// Warning (FieldDecl): 5859: function pointers are not supported [1] : 'int (*)(sqlite3 *, void *, int, char **, sqlite3_vtab **, char **)'
// Warning (FieldDecl): 5862: function pointers are not supported [1] : 'int (*)(sqlite3 *, void *, int, char **, sqlite3_vtab **, char **)'
// Warning (FieldDecl): 5865: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *, sqlite3_index_info *)'
// Warning (FieldDecl): 5866: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *)'
// Warning (FieldDecl): 5867: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *)'
// Warning (FieldDecl): 5868: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *, sqlite3_vtab_cursor **)'
// Warning (FieldDecl): 5869: function pointers are not supported [1] : 'int (*)(sqlite3_vtab_cursor *)'
// Warning (FieldDecl): 5870: function pointers are not supported [1] : 'int (*)(sqlite3_vtab_cursor *, int, char *, int, sqlite3_value **)'
// Warning (FieldDecl): 5872: function pointers are not supported [1] : 'int (*)(sqlite3_vtab_cursor *)'
// Warning (FieldDecl): 5873: function pointers are not supported [1] : 'int (*)(sqlite3_vtab_cursor *)'
// Warning (FieldDecl): 5874: function pointers are not supported [1] : 'int (*)(sqlite3_vtab_cursor *, sqlite3_context *, int)'
// Warning (FieldDecl): 5875: function pointers are not supported [1] : 'int (*)(sqlite3_vtab_cursor *, sqlite3_int64 *)'
// Warning (FieldDecl): 5876: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *, int, sqlite3_value **, sqlite3_int64 *)'
// Warning (FieldDecl): 5877: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *)'
// Warning (FieldDecl): 5878: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *)'
// Warning (FieldDecl): 5879: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *)'
// Warning (FieldDecl): 5880: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *)'
// Warning (FieldDecl): 5881: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *, int, char *, void ( **)(sqlite3_context *, int, sqlite3_value **), void **)'
// Warning (FieldDecl): 5884: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *, char *)'
// Warning (FieldDecl): 5887: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *, int)'
// Warning (FieldDecl): 5888: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *, int)'
// Warning (FieldDecl): 5889: function pointers are not supported [1] : 'int (*)(sqlite3_vtab *, int)'

```

**Comment by Konstantin8105** [comment]
Check after #490

**Comment by Konstantin8105** [comment]
```c
struct sqlite3_module {
  int iVersion;
  int (*xCreate)(sqlite3*, void *pAux,
               int argc, const char *const*argv,
               sqlite3_vtab **ppVTab, char**);
  int (*xConnect)(sqlite3*, void *pAux,
               int argc, const char *const*argv,
               sqlite3_vtab **ppVTab, char**);
  int (*xBestIndex)(sqlite3_vtab *pVTab, sqlite3_index_info*);
  int (*xDisconnect)(sqlite3_vtab *pVTab);
  int (*xDestroy)(sqlite3_vtab *pVTab);
  int (*xOpen)(sqlite3_vtab *pVTab, sqlite3_vtab_cursor **ppCursor);
  int (*xClose)(sqlite3_vtab_cursor*);
  int (*xFilter)(sqlite3_vtab_cursor*, int idxNum, const char *idxStr,
                int argc, sqlite3_value **argv);
  int (*xNext)(sqlite3_vtab_cursor*);
  int (*xEof)(sqlite3_vtab_cursor*);
  int (*xColumn)(sqlite3_vtab_cursor*, sqlite3_context*, int);
  int (*xRowid)(sqlite3_vtab_cursor*, sqlite3_int64 *pRowid);
  int (*xUpdate)(sqlite3_vtab *, int, sqlite3_value **, sqlite3_int64 *);
  int (*xBegin)(sqlite3_vtab *pVTab);
  int (*xSync)(sqlite3_vtab *pVTab);
  int (*xCommit)(sqlite3_vtab *pVTab);
  int (*xRollback)(sqlite3_vtab *pVTab);
  int (*xFindFunction)(sqlite3_vtab *pVtab, int nArg, const char *zName,
                       void (**pxFunc)(sqlite3_context*,int,sqlite3_value**),
                       void **ppArg);
  int (*xRename)(sqlite3_vtab *pVtab, const char *zNew);
  /* The methods above are in version 1 of the sqlite_module object. Those 
  ** below are for version 2 and greater. */
  int (*xSavepoint)(sqlite3_vtab *pVTab, int);
  int (*xRelease)(sqlite3_vtab *pVTab, int);
  int (*xRollbackTo)(sqlite3_vtab *pVTab, int);
};
```

**Comment by elliotchance** [comment]
Was this resolved with #490?
